packages/spaces/src/helpers.js

```diff
@@ -71,3 +73,64 @@ export const updateUrl = (url, key, value) => {
 
   return `${uri}?${newParams}`;
 };
+
+/**
+ * Top Apps
+ */
+
+const TOP_APPS = {
+  ALLOWED_TYPES: ['APPLICATION', 'RESOURCE', 'NAVIGATION'],
+  BLACKLIST: ['reporting', 'how_to_guide_dental_providers', 'my_account_profile', 'my_administrators'],
+  KEYS: {
+    LAST_UPDATED: 'top-apps-updated',
+    VALUES: 'myTopApps',
+  },
+  UPDATE_EVENT: 'av:topApps:updated',
+};
+
+const getItemLocalStorage = (key) => {
+  const value = window.localStorage.getItem(key);
```

Comment:
`localStorage.getItem()` will return `null` if the item doesn't exist, and we should be able to return the value without creating an intermediate variable. i think we can change the function to something like this:

```js
const getItemLocalStorage = (key) => {
  try {
    const value = window.localStorage.getItem(key);
    
    // avoids JSON.parse() overhead if value already null
    if(value === null) {
      return null;
    }
    return JSON.parse(value);
  } catch {
    return null;
  }
};
```

Reply:
Nice catch, yeah that's a good idea